### ISSUE TITLE
Basic usage not work

### ISSUE THREAD

**Issue by ygj0930** [tree-sitter/kotlin-tree-sitter#32]
val language = Language(TreeSitterKotlin.language())
val parser = Parser(language)
val tree = parser.parse("fun main() {}")
val rootNode = tree.rootNode

But  TreeSitterKotlin.language()  not exist?

**Comment by sdfgsdfgd** [comment]
that means your library doesn't work then, why call it a tree sitter kotlin lib ?


tree sitter by default, on python, is bound to multiple language grammars and parsers which are statically linked as you enable them via a simple call ! 

If you're going to write a kotlin library for it, it needs to work out of the box with parsers. What is the point of uploading a pile of  ***** 

Pls remove this repository from github and upload it when you're done

nobody is going to do your job for you and link to the shared libraries and grammar .so  files

**Comment by ObserverOfTime** [comment]
> tree sitter by default, on python, is bound to multiple language grammars and parsers which are statically linked as you enable them via a simple call

No, it is not. If you're talking about `tree-sitter-languages`, that is unofficial and outdated.

**Comment by chetanIxigo** [comment]
`TreeSitterKotlin` is unresolved. why is that?

**Comment by marko88** [comment]
How to use them and how to parse kotlin code?

**Comment by sdfgsdfgd** [comment]
ye how do we parse kotlin code into nodes do we have to hunt for grammar rule files before we can even try ? why isnt there a doc
